meta-mel/conf/distro/mel.conf

```diff
@@ -232,9 +240,6 @@ BUILDHISTORY_COMMIT ?= "1"
 # Copy the image license manifest into DEPLOY_DIR_IMAGE
 INHERIT += "deploy-license-manifest"
 
-# Support USER_FEATURES
-INHERIT += "user_features"
```

Comment:
ah ok so it was moved but in the same commit?

Reply:
Yes, just rearranged because we have a number of cases where we provide mechanisms to make certain things easier from local.conf for the customer.